### PR TITLE
Exercises 1 / 2 / 3

### PR DESCRIPTION
**Pull request by bchoplin**
as titled 🙂

### CODE REVIEW

File: src/exercise/03.html [hunk 1]
```diff
@@ -6,26 +6,17 @@
...
+    const className = 'container';
+    const props = {children, className};
+    // const element = <div {...props} />;
 
-    // 🐨 re-implement this using JSX!
-    const element = React.createElement('div', {
-      className: 'container',
-      children: 'Hello World',
-    })
+    // writing it this way for readability / alley standards
```
Comment (thischrisblack) [inline src/exercise/03.html:17]: info: Do you mean "this way" as opposed to the `...props` syntax?

**Review by thischrisblack — APPROVED** [review]
🐴

File: src/exercise/03.html (hunk 1 above)
Comment (bchoplin) [reply src/exercise/03.html:17]: yep! travis did something similar in his PR because he said `...props` gets rejected in CR a lot

Comment (mslinnea) [reply src/exercise/03.html:17]: Interesting!

**Pull request merged**